app/helpers/application_helper.rb

```diff
@@ -163,10 +163,6 @@ def logo_svg
     end
   end
 
-  def safe_logo_url(logo)
```

Comment:
This helper was only used to wrap `SiteConfig.secondary_logo` so it can be removed now. All call sites have been changed to directly use `SiteConfig.logo_png`.